.github/workflows/insiders-fast.yml

```diff
@@ -27,7 +28,9 @@ jobs:
   publish:
     name: Publish
     needs: changes
-    if: ${{ needs.changes.outputs.new == 'true' }}
+    if: |
+      always() &&
```

Comment:
No it doesn't because we are chaining the conditional, its needed because the of the `needs`  job in the dependency chain. Because its skipped, the `always()` is necessary for it to execute. It seems odd but see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds